src/__swaps__/utils/swaps.ts

```diff
@@ -46,6 +46,8 @@ import { IS_APK_BUILD } from 'react-native-dotenv';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import isTestFlight from '@/helpers/isTestFlight';
 
+const DEFAULT_SLIPPAGE_BIPS = 500;
```

Comment:
I think we should share this constant with the `remoteConfig` so that changing one changes both?

Reply:
I agree but this needs more work so whenever I have time I'm gonna do this:

https://linear.app/rainbow/issue/APP-2286/clean-up-slippage-defaults